src/Caching/StackExchangeRedis/src/StackExchangeRedisCacheServiceCollectionExtensions.cs

```diff
@@ -12,6 +12,16 @@ namespace Microsoft.Extensions.DependencyInjection;
 /// </summary>
 public static class StackExchangeRedisCacheServiceCollectionExtensions
 {
+    /// <summary>
+    /// Adds Redis distributed caching services to the specified <see cref="IServiceCollection" />.
+    /// </summary>
+    /// <param name="services">The <see cref="IServiceCollection" /> to add services to.</param>
+    /// <returns>The <see cref="IServiceCollection"/> so that additional calls can be chained.</returns>
+    public static IServiceCollection AddStackExchangeRedisCache(this IServiceCollection services)
+    {
+        return AddStackExchangeRedisCache(services, (option) => { });
```

Comment:
Let's try to follow the pattern used by other extension methods where we set up the services etc. in the non-action method and call this one from the action method
https://github.com/dotnet/aspnetcore/blob/main/src/Middleware/CORS/src/CorsServiceCollectionExtensions.cs